src/binary-reader-objdump.cc

```diff
@@ -57,12 +60,18 @@ class BinaryReaderObjdumpBase : public BinaryReaderNop {
   bool print_details = false;
   BinarySection reloc_section = BinarySection::Invalid;
   uint32_t section_starts[kBinarySectionCount];
+  int section_found = false;
+  bool header_printed = false;
```

Comment:
Not strictly related to this change, but is `header_printed` actually used anywhere? It was a bit confusing to have both `header_printed` and `options->print_header`, so maybe we can get rid of both?